nemo/utils/exp_manager.py

```diff
@@ -238,6 +280,11 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
             cfg.wandb_logger_kwargs,
         )
 
+    # add loggers timing callbacks
+    if cfg.log_step_timing:
+        timing_callback = TimingCallback()
```

Comment:
No argument sync_cuda passed to the NamedTimer

Reply:
Fixed.